sql/src/test/java/io/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -4928,9 +4994,15 @@ public void testFilterOnTimeExtract() throws Exception
         ImmutableList.of(
             Druids.newTimeseriesQueryBuilder()
                   .dataSource(CalciteTests.DATASOURCE1)
-                  .intervals(QSS(Intervals.of("2000/P1M")))
+                  .intervals(QSS(Intervals.of("-146136543-09-08T08:23:32.096Z/146140482-04-24T15:36:27.903Z")))
```

Comment:
This would be better as `.intervals(QSS(Filtration.eternity()))`

Reply:
thanks, changed to Filtration.eternity()